airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -22,3 +22,4 @@ tests:
       skip_comprehensive_incremental_tests: true
   full_refresh:
     - config_path: "secrets/config.json"
+# TODO remove this change
```

Comment:
This change was only made to check the GitHub workflows works as expected. Must be reverted before merge.